packages/wdio-cli/src/commands/install.js

```diff
@@ -1,36 +1,33 @@
 /* eslint-disable no-console */
 import fs from 'fs'
 import path from 'path'
-import inquirer from 'inquirer'
 import yarnInstall from 'yarn-install'
-/**
-    wdio install <type> <packageName>
- */
-import setup from '../setup'
-import { SUPPORTED_SERVICES, SUPPORTED_REPORTER, SUPPORTED_FRAMEWORKS } from '../config'
+
 import {
-    parseInstallNameAndPackage,
     replaceConfig,
     findInConfig,
-    addServiceDeps
+    addServiceDeps,
+    convertPackageHashToObject,
+    missingConfigurationPrompt
 } from '../utils'
 
+import { SUPPORTED_PACKAGES } from '../constants'
+
 const supportedInstallations = {
-    service: parseInstallNameAndPackage(SUPPORTED_SERVICES),
-    reporter: parseInstallNameAndPackage(SUPPORTED_REPORTER),
-    framework: parseInstallNameAndPackage(SUPPORTED_FRAMEWORKS)
+    service: SUPPORTED_PACKAGES.service.map(({ value }) => convertPackageHashToObject(value)),
+    reporter: SUPPORTED_PACKAGES.reporter.map(({ value }) => convertPackageHashToObject(value)),
+    framework: SUPPORTED_PACKAGES.framework.map(({ value }) => convertPackageHashToObject(value))
 }
 
 export const command = 'install <type> <name>'
 export const desc = 'Add a `reporter`, `service`, or `framework` to your WebdriverIO project'
 
-export default function builder(yargs) {
-    return yargs
-        .option('npm', {
-            desc: 'Install packages using npm',
-            type: 'boolean',
-            default: false
-        })
+export const builder = {
+    npm: {
+        desc: 'Install packages using npm',
+        type: 'boolean',
+        default: false
```

Comment:
Does it mean that `yarn` is treated as default package manager?
If so I'd change it to `true`.
I think not many users have yarn, especially new comers.

Reply:
Yea, that's what it means. I'll change it.

Reply:
I would keep NPM as default package manager.

Reply:
Updated: 766f80f